src/server/redis_connection.cc

```diff
@@ -397,8 +397,15 @@ void Connection::ExecuteCommands(std::deque<CommandTokens> *to_process_cmds) {
 
     const auto &attributes = current_cmd->GetAttributes();
     auto cmd_name = attributes->name;
-    auto cmd_flags = attributes->GenerateFlags(cmd_tokens);
```

Comment:
We should also check this call in scripting, not only here.

Reply:
Good point, done.